tests/e2e/kubernetes/kubernetes_test.go

```diff
@@ -14,8 +14,8 @@ import (
 )
 
 const (
-	currentRuntimeVersion   = "1.2.0"
-	currentDashboardVersion = "0.6.0"
+	currentRuntimeVersion   = "1.3.0"
```

Comment:
@mukundansundar I see the duplication of versioning information across various files. Do we need to refactor it a little bit?

Reply:
@anhldbk yes ... I think minor refactoring to remove duplicate versions in tests can be done as a separate PR. This PR cannot be changed ..